swift/ios-shoppe-demo/ServiceManagers/FullStoryManager.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  FullStoryManager.swift
+//  ios-shoppe-demo
+//
+//  Created 5/7/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
+import Foundation
+import FullStory
+
+/**
+ FullStory Manager is a singleton used to manage api calls to FullStory.
+
+    - create events
+    - identify a user masking and unmasking elements
+ */
+class FullStoryManager {
+    static var shared = FullStoryManager()
+
+    enum Event: String {
+        case browsing
+        case itemSelected
+        case viewCart
+        case addToCart
+        case checkout
+        case removeFromCart
+        case crash
+    }
+
+    enum LogLevel {
+        case assert, error, warning, info, debug
+    }
+
+    enum FSStatus: String {
+        case exclude = "fs-exclude"
+        case mask = "fs-mask"
+        case unmask = "fs-unmask"
+    }
+
+    var currentEvent: Event = .browsing
```

Comment:
I'm not clear why this needs to be a member.

Reply:
To be a member? I'm not sure I understand. but I think for now there is not a real need for currentEvent.